adagrams/game.py

```diff
@@ -1,11 +1,104 @@
+from string import ascii_uppercase
+from random import randint , choice
+
+
+
 def draw_letters():
-    pass
+    letter_pool = {
+        'A': 9, 
+        'B': 2, 
+        'C': 2, 
+        'D': 4, 
+        'E': 12, 
+        'F': 2, 
+        'G': 3, 
+        'H': 2, 
+        'I': 9, 
+        'J': 1, 
+        'K': 1, 
+        'L': 4, 
+        'M': 2, 
+        'N': 6, 
+        'O': 8, 
+        'P': 2, 
+        'Q': 1, 
+        'R': 6, 
+        'S': 4, 
+        'T': 6, 
+        'U': 4, 
+        'V': 2, 
+        'W': 2, 
+        'X': 1, 
+        'Y': 2, 
+        'Z': 1
+    }
+    hand = []
+
+
+    while len(hand) < 10:
```

Comment:
I'd like to see the literal integer `10` referenced by a constant variable. As someone who might not know anything about this game, I might read line 38 and wonder what the significance of 10 means. 

How about something like:
```python
HAND_SIZE = 10
while len(hand) < HAND_SIZE:
```

Reply:
Thank you, I'll fix that

Reply:
Done